content/usage/advanced/index.md

```diff
@@ -444,17 +795,172 @@ relevant "Joystick mapping", from the button IDs presented by the physical joyst
+
+{{ easy_image(src="dev-mode-config", width=600) }}
+
+- **Development mode** turns on a debugging overlay that displays relevant information about the visible widgets
+   - Dev info blur level determines how much the underlying interface is blurred when in dev mode
+- **BlueOS settings sync** allows synchronising Cockpit's settings to the connected vehicle (if it is running BlueOS)
+   - This helps with keeping the same settings and profiles if that vehicle is then later controlled using
+     a different device running Cockpit
+- **Usage statistics telemetry** can be automatically sent to the Cockpit development team to help identify and
+  track common errors and performance issues across global Cockpit usage
+- **System logging** creates [application logs](#system-application-logs) that can be checked or shared with
+  developers to help debug issues
+   - If system logs are disabled then messages are sent to the browser/application console instead, which may
+     reduce performance
+
+### Missions and Safety
+
+While autopilots often include built in failsafes and pre-arming checks, it can also be useful for the operator's
+interface to require explict confirmation before a safety-related or mission critical action is even sent to the
+vehicle.
+
+Depending on the interface configuration, some interfaces may make it easy to accidentally press a button or
+slider on the screen that has an undesirable outcome, so it is possible to require these actions to be have an
+extra confirmation step:
+
+{{ easy_image(src="mission-config", width=500) }}
+
+A confirmation slider is shown, which can either be dragged to completion in the interface, or confirmation can
+be provided by pressing and holding a joystick button that is configured to the `hold_to_confirm` Action:
+
+{{ easy_image(src="slide-to-confirm", width=250, center=true) }}
+
+{% note() %}
+Actions that are *triggered* by joystick button presses (instead of interface elements) do not require extra
+confirmation - it is assumed that the press of a dedicated button is enough confirmation.<br><br>
+If you want a similar feature for joystick button functions, consider assigning them to a
+[modifier-based slot](#modifier-keys), to require a modifier key to be held down before the function
+can be triggered.
+{% end %}
+
+### Cockpit Actions
+
+Cockpit's Action system provides a set of functionalities that can be triggered from any of Cockpit's
+supported input sources (e.g. joystick buttons, interface elements, other Actions, etc).
+
+#### Default Actions
+There are some predefined Actions built into Cockpit for convenience, including:
+
+- `go_to_next_view`
+- `go_to_previous_view`
+- `toggle_bottom_bar`
+- `toggle_full_screen`
+- `hold_to_confirm`
+- `start_recording_all_streams`
+- `stop_recording_all_streams`
+- `mavlink_arm`
+- `mavlink_disarm`
+
+#### Custom Actions
+It is also possible to define (and export or import) your own custom Actions, with a few different approaches:
```

Comment:
It would be interesting to mention where the data-lake variables can be used on those actions (anywhere in a javascript action, on the parameters and body of a http action or in the message of a mavlink action).